server/internal/light/components.go

```diff
@@ -297,6 +302,7 @@ func newComponents(ctx context.Context, cfg configuration.Configuration) (*compo
 	}
 
 	c.cmp.Inject(
+		WmBus,
```

Comment:
It doesn't (and shouldn't) have any injects. No need to pass it here.

Reply:
I want current MessageBus to inject Bus, so MessageBus can be wrapper over Send(...)<-msg method